docusaurus/docs/frontend.md

```diff
@@ -3,7 +3,7 @@ id: frontend
 title: Frontend
 ---
 
-The Frontend part of Grafana plugins are written in Typescript with support for React as the view library. Within the plugin directory, you can run:
+The frontend of Grafana plugins are written in Typescript with React as the view library. Within the plugin directory, you can run:
```

Comment:
"frontend of Grafana plugins" doesn't quite flow on account of shift in plurals. Would 'Frontends for Grafana plugins..." work better?